app/src/androidTest/java/com/conupods/IndoorMaps/ConcreteBuilding.java

```diff
@@ -0,0 +1,78 @@
+package com.conupods.IndoorMaps;
+
+import com.conupods.IndoorMaps.ConcreteBuildings.CCBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.VLBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.MBBuilding;
+import com.conupods.IndoorMaps.ConcreteBuildings.HBuilding;
+import com.conupods.OutdoorMaps.Models.Building.Building;
+import com.google.android.gms.maps.model.LatLng;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConcreteBuilding {
```

Comment:
change it to define it is a test

Reply:
that isn't necessary as the methods are classified